.appveyor.yml

```diff
@@ -101,6 +103,8 @@ install:
   # Upgrade to the latest pip.
   - 'python -m pip install -U pip setuptools wheel'
 
+  - if [%INSTALL_PICKLE5%]==[1] echo pickle5 >> tools/ci/appveyor/requirements.txt
```

Comment:
Can the changes to testing infrastructure be broken out into a  separate PR? We also added azure to the mix there for windows and mac

Reply:
It's possible but this will make the coverage statistics of this specific PR drop significantly and make it harder to check that this PR does not introduce too much uncovered code.

Reply:
I don't follow. Even if coverage drops, it will do so whether this PR is broken into two or remains as one, no?

Reply:
But maybe you mean to have one PR on top of the other then? Alright then.

Reply:
So what I understand is that I should keep this PR unchanged, while copying the changes of the CI suite to another PR (same for `numpy.dump`/`numpy.dumps` ). 

Chronologically speaking, We will then merge the two small PRs first, and then this one.

Reply:
Yeah, appveyor isn't being used now & we don't use that requirements file to parse dependencies on windows at the moment, though we could probably look into doing that again if needed.